### DISCUSSION TITLE
Wing drag minimization example problem

### DISCUSSION THREAD

**Discussion by Airbreathing** [peterdsharpe/AeroSandbox#76] [Q&A] [answered]
Hi!
I have a question on the wing drag minimization example problem (https://github.com/peterdsharpe/AeroSandbox/blob/master/tutorial/02%20-%20Design%20Optimization/02%20-%20Wing%20Drag%20Minimization%2C%20with%20practical%20considerations.ipynb). I was wondering why the take-off speed is just an input and it is not allowed to vary? It could be set to 1.2*V_stall, where V_stall depends on the design variables aircraft weight and wing area.
I noticed they do a similar thing in the Master's thesis referenced here: https://github.com/peterdsharpe/AeroSandbox/blob/master/tutorial/02%20-%20Design%20Optimization/03%20-%20Aircraft%20Design%20-%20SimPleAC.ipynb. Should that take-off speed be regarded as a minimum velocity and be calculated based on the initial aircraft parameters? That could explain why it is set as an input and not allowed to vary.
Thank you in advance!

**Comment by peterdsharpe** [comment, marked as answer]
Hi there!

Great question! As you noted, the problem in the notebook was implemented following Berk's thesis. The original formulation I think goes back to [Woody Hoburg](https://en.wikipedia.org/wiki/Warren_Hoburg)'s PhD thesis, [available here](https://web.mit.edu/~whoburg/www/papers/hoburg_phd_thesis.pdf)! 

In short, it's arbitrary - the minimum velocity could easily be allowed to vary, if that's of interest! Feel free to change it to a variable, set the appropriate constraints, and see how the design changes :)

**Reply by Airbreathing** [reply]
Got it! Thank you for your answer (and for your great work!) :)

**Comment by dangraf** [comment]
Hello!
I'm looking in to your fantastic library and have a pretty basic question since I have not studied aerodynamics in full.
This example of yours is very interesting for me (https://github.com/peterdsharpe/AeroSandbox/blob/master/tutorial/06%20-%20Aerodynamics/01%20-%20AeroSandbox%203D%20Aerodynamics%20Tools/01%20-%20Vortex%20Lattice%20Method.ipynb)
But where can I find the documentation of what the output of the simulation are?

   D : 1.5500742218811423
   Y : 7.706855400298031e-16
   l : -2.839478665173283e-15
   m : -2.255208490849847
   n : 6.874722327848233e-16
  CL : 0.7166310680840677
  CD : 0.013849141449451634
  CY : 6.8856916051199124e-18
  Cl : -1.256025607871831e-17
  Cm : -0.13339717342124371
  ....
  I can guess some of them like Cl , D etc but is it possible to get a full list of all parameters and the units they are presented as in the documentation somewhere?
  
  thanks!

**Reply by peterdsharpe** [reply]
Sure! [They are here](https://github.com/peterdsharpe/AeroSandbox/blob/013d36fe79fe3b437e4bc2137599559b1e4f51af/aerosandbox/aerodynamics/aero_3D/vortex_lattice_method.py#L84).

One particular note of caution is that `Cl` and `CL` are not the same in aerodynamics and flight dynamics - `Cl` is the rolling moment coefficient, and `CL` is the lift coefficient.

For more details on coordinate systems, conventions, and aerodynamics basics, I'd highly recommend [Flight Vehicle Aerodynamics by Mark Drela](https://smile.amazon.com/Flight-Vehicle-Aerodynamics-MIT-Press/dp/0262526441?sa-no-redirect=1).
